pandas/io/json/_json.py

```diff
@@ -514,6 +514,7 @@ def read_json(
         - ``'index'`` : dict like ``{{index -> {{column -> value}}}}``
         - ``'columns'`` : dict like ``{{column -> {{index -> value}}}}``
         - ``'values'`` : just the values array
+        - None : orient will then be assigned based on `typ` parameter
```

Comment:
This seems to be redundant with the next paragraph, where it's explained what the default value of orient will be based on `typ`.  And IMHO, it makes things more confusing. I'd just remove this line and just leave the existing paragraph.

Reply:
After reading it again, I agree with you completely. Removing it!!